model/metrics/dotnet/dotnet-health_checks.yaml

```diff
@@ -0,0 +1,49 @@
+groups:
+  - id: health_checks
+    prefix: health_checks
+    type: attribute_group
+    brief: Health Check attributes
+    attributes:
+      - id: dotnet.health_check.status
+        type:
+          allow_custom_values: true
+          members:
+            - id: degraded
+              value: 'Degraded'
+              brief: "An application was in degraded state."
+            - id: healthy
+              value: 'Healthy'
+              brief: "An application was healthy."
+            - id: unhealthy
+              value: 'Unhealthy'
+              brief: "An application was unhealthy."
+        brief: The health status of an application.
+        examples: ["Healthy", "Unhealthy"]
+        requirement_level: required
+
+  - id: metric.dotnet.health_check.reports
```

Comment:
is it fair to say that `http.server.request.duration` for specific health-check route would contain the same information? 
Also, would it make sense to make it duration histogram (and derive count from it?